packages/xpetra/src/Map/Xpetra_TpetraMap_def.hpp

```diff
@@ -300,7 +298,8 @@ RCP< const Tpetra::Map< LocalOrdinal, GlobalOrdinal, Node > > TpetraMap<LocalOrd
 #ifdef HAVE_XPETRA_TPETRA
 
 template<class LocalOrdinal, class GlobalOrdinal, class Node>
-local_map_type TpetraMap<LocalOrdinal, GlobalOrdinal, Node>::getLocalMap () const
+//local_map_type TpetraMap<LocalOrdinal, GlobalOrdinal, Node>::getLocalMap () const
```

Comment:
You could just delete the line of code, since it's most definitely incorrect syntax :)

Reply:
heh, forgot to remove that commented out line.  It'll be gone next time I push to my branch.  I'm trying to track down some other errors right now that aren't clear to me if they're from my code or if they're something else and I just happened to hit them because of what I've turned on in CMake... /shrug